packages/js/src/components/metabox-error-fallback.js

```diff
@@ -0,0 +1,49 @@
+import PropTypes from "prop-types";
+import { useSelect } from "@wordpress/data";
+import { useCallback, useEffect } from "@wordpress/element";
+import { disableMetaboxTabs } from "../helpers/disableMetaboxTabs";
+import ScoreIconPortal from "./portals/ScoreIconPortal";
+import { Root } from "@yoast/ui-library";
+import { ErrorFallback } from "../shared-admin/components/error-fallback";
+import isInclusiveLanguageAnalysisActive from "../analysis/isInclusiveLanguageAnalysisActive";
+
+/**
+ * @param {Object} error The error instance.
+ * @returns {JSX.Element} The error fallback element.
+ */
+export const MetaboxErrorFallback = ( { error } ) => {
+	const handleRefreshClick = useCallback( () => window?.location?.reload(), [] );
+	const supportLink = useSelect( select => select( "yoast-seo/editor" ).selectLink( "https://yoa.st/metabox-error-support" ), [] );
+	const isRtl = useSelect( select => select( "yoast-seo/editor" ).getPreference( "isRtl", false ), [] );
+
+	useEffect( () => {
+		disableMetaboxTabs();
+	}, [] );
+
+	return (
+		<Root context={ { isRtl } }>
+			<ErrorFallback error={ error }>
+				<ErrorFallback.HorizontalButtons
+					supportLink={ supportLink }
+					handleRefreshClick={ handleRefreshClick }
+				/>
+				<ScoreIconPortal
+					target="wpseo-readability-score-icon"
+					scoreIndicator={ "not-set" }
+				/>
+				<ScoreIconPortal
+					target="wpseo-seo-score-icon"
+					scoreIndicator={ "not-set" }
+				/>
+				{ isInclusiveLanguageAnalysisActive && <ScoreIconPortal
```

Comment:
Question, is there a reason that for inclusive language we check if the feature is enabled, but for eg. readability we don't?

For uniformity purposes, shouldnt have this the same?

Reply:
Yes, I removed it.